discord_bot.py

```diff
@@ -2,8 +2,38 @@
 
 from discord.ext import commands
 
+from cogs.poll import get_new_description, string_description_to_dictionary, generate_embed
```

Comment:
This whole set of changes in this file feels odd to me. Adding specific actions for a specific cog/command to the core bot seems like a antipattern. Shouldn't we think of away to have these act as hooks that we can hook into the on_raw_reaction_add and remove events rather than adding specific cases for a specific cog/command to the base event?

Reply:
I've searched for quite a while. Went on their discord and asked of ways to accomplish this and this is the only way i found that works. Thats the same way they said is "best practice" in terms of discord bot. If you know another way I'd gladly try it. Just give me a bit more information in terms of code.

Reply:
Okay, if this is the best way you've found so far, then I'm fine with this way, I'll add something to the backlog on Trello to investigate alternatives at some point.

Reply:
Will also make a github issue for it